build.cmd

```diff
@@ -55,6 +55,8 @@ set Platform=
 :: Log build command line
 set _buildproj=%~dp0build.proj
 set _buildlog=%~dp0msbuild.log
+set _binclashLoggerDll=%~dp0Tools\net45\Microsoft.DotNet.Build.Tasks.dll
```

Comment:
This is making the assumption that we are running on top of net45. I think that is reasonable at this point but something we need to keep in mind.